server/chat-plugins/thing-of-the-day.ts

```diff
@@ -187,7 +187,7 @@ class OtdHandler {
 
 		const nomObj = {
 			nomination: nomination,
-			name: user.name,
+			name: toID(user),
```

Comment:
This shouldn't be a user ID - you should just escape it with `Utils.escapeHTML` (either here or when it's displayed).